iFixit/react-commerce
# Add page template quote gallery section

closes #1559 

❗❗❗After this PR is merged Strapi should be redeployed in production ❗❗❗

cc @sterlinghirsh @danielbeardsley @masonmcelvain 

## QA

1. Open [store home page preview](https://react-commerce-git-add-page-template-quote-galler-88d402-ifixit.vercel.app/store)
2. Verify the the quote gallery section is rendered (it should look like #1559 but with @mmarcon91 we made a few changes mainly on the size of the cards, so just check that it looks fine and works fine)
3. Verify that the section is responsive
4. Verify that you can change section content from [Strapi](https://add-page-template-quote-gallery-section.govinor.com/admin/content-manager/collectionType/api::page.page/1?plugins[i18n][locale]=en) (we changed Strapi password, DM me or Mason in Slack to get the new password)
